source/server/management/exploit-patches.rst

```diff
@@ -0,0 +1,28 @@
+===============
+Exploit Patches
+===============
+
+In recent Sponge builds (SpongeForge 974+), Sponge patches a few client-server exploits. Whenever Sponge detects a
+user performing an exploit, they are kicked from the server with a message explaining why they were kicked. If enabled,
+a log message is also sent to the console. More exploit patches may be added in the future.
+
+Exploits Patched by Sponge
+==========================
+
+#. Sign command exploit where a client could run a command such as 'op'
+#. Client could force the server to make the user respawn invisible
+#. Client could set an itemstack's display name and cause it to exceed the character limit
+
+.. warning::
+    The invisibility exploit has been disabled in recent Sponge builds due to false triggering.
```

Comment:
so the exploit cannot be used? or the fix for the exploit has been disabled? Also please clarify the "false triggering"